modules/virt-viewing-virtual-machine-logs-web.adoc

```diff
@@ -5,7 +5,7 @@
 [id="virt-viewing-virtual-machine-logs-web_{context}"]
 = Viewing virtual machine logs in the web console
 
-Get virtual machine logs from the associated virtual machine launcher Pod.
+Get virtual machine logs from the associated virtual machine launcher pod.
 
 .Procedure
 
```

Comment:
(Below) s/. In the *Details* tab, click the `virt-launcher-<name>` Pod/. In the *Details* tab, click the `virt-launcher-<name>` pod/